src/main/java/com/dreammaster/recipes/RecipeRemover.java

```diff
@@ -2194,6 +2195,64 @@ public static void run() {
         removeRecipeByOutputDelayed(getModItem(ProjectRedCore.ID, "projectred.core.screwdriver", 1, 0, missing));
         removeRecipeByOutputDelayed(getModItem(ProjectRedCore.ID, "projectred.core.wiredebugger", 1, 0, missing));
         removeRecipeByOutputDelayed(getModItem(ProjectRedCore.ID, "projectred.core.datacard", 1, 0, missing));
+        removeRecipeByOutputDelayed(getModItem(ProjectRedIntegration.ID, "projectred.integration.gate", 1, 0, missing));
```

Comment:
better would be to stop them from being added in projectred (if nhcoremod is loaded). but the mod is in scala, so I understand if you rather not.